server/index.js

```diff
@@ -5,11 +5,13 @@ const port = 3000;
 const app = express();
 
 // For dev purposes
-// const morgan = require('morgan');
```

Comment:
I've seen morgan and dev used in my code as well, not sure exactly what they do but know its related to development. I'll have to do research to see what their purpose is